mod/thewire/views/default/thewire.js

```diff
@@ -27,15 +27,16 @@ elgg.thewire.textCounter = function(textarea, status, limit) {
 
 	var remaining_chars = limit - $(textarea).val().length;
 	status.html(remaining_chars);
+	var $submit = $(textarea).closest('.elgg-form-thewire-add').find('#thewire-submit-button').eq(0);
```

Comment:
Do you expect the form to contain multiple submit buttons with the same id? I don't think `.eq()` is needed here.